include/http/request_handler.js

```diff
@@ -603,6 +600,8 @@ module.exports = function RequestHandlerModule(pb) {
                 pb.log.warn("RequestHandler: The active theme is not set.  Defaulting to '%s'", RequestHandler.DEFAULT_THEME);
                 activeTheme = RequestHandler.DEFAULT_THEME;
             }
+            //get locale preference
+            self.localizationService = new pb.Localization(self.req, activeTheme);
```

Comment:
You will have to initialize the localization service sooner.  The implementation here will cause an error when a non-existent route is provided.  See line ~590.  

I think what you need is a setter on the Localization service that allows you to set the prioritized theme.  By default, or when provided null or undefined it would default to ```pencilblue```.